docroot/themes/humsci/humsci_basic/src/scss/components/_main-nav.scss

```diff
@@ -171,6 +179,7 @@
 
     &:hover:not(.is-active) {
```

Comment:
hmm we might need a focus style here too?

Reply:
See line 224. Adding the focus state here darkens the top level nav items. I `focused` on the dropdown nav items. :smile: